keras/backend/jax/numpy.py

```diff
@@ -322,6 +323,7 @@ def average(x, axis=None, weights=None):
 
 
 def broadcast_to(x, shape):
+    x = convert_to_numpy(x)
```

Comment:
Use `x = convert_to_tensor(x)`

Reply:
Thank you!

Reply:
@fchollet ,Thanks for the input. Changes done as suggested and works fine.